dcm2niix_gear/utils/resolve.py

```diff
@@ -176,8 +177,10 @@ def retain_gear_outputs(
 
             if retain_nifti:
                 # If "left" over is simply the extension, this image matches
-                #   the current sidecar, move to output.
-                if left in [".nii.gz", ".nii", ".bval", ".bvec"]:
+                #   the current sidecar, move to output. If an underscore is in "left", then
+                # there is likely a longer string filename that will match a specific sidecar
+                # in a future loop. `and left.startswith('.') may be equally as useful
+                if (left in [".nii.gz", ".nii", ".bval", ".bvec"]) and (not "_" in left):
```

Comment:
I just don't see how this changes anything.  If `left` is in that list, then it wouldn't contain a `"_"` since none of the values in the list contain one. Maybe I'm not understanding the original issue.